Code/Framework/AzFramework/AzFramework/Physics/SimulatedBodies/StaticRigidBody.h

```diff
@@ -32,12 +32,20 @@ namespace AzPhysics
     {
     public:
         AZ_CLASS_ALLOCATOR_DECL;
-        AZ_RTTI(StaticRigidBody, "{13A677BB-7085-4EDB-BCC8-306548238692}", SimulatedBody);
+        AZ_RTTI(AzPhysics::StaticRigidBody, "{13A677BB-7085-4EDB-BCC8-306548238692}", AzPhysics::SimulatedBody);
         static void Reflect(AZ::ReflectContext* context);
 
-        //Legacy API - may change with LYN-438
+        //! Add a shape to the static rigid body.
+        //! @param shape A shared pointer of the shape to add.
         virtual void AddShape(const AZStd::shared_ptr<Physics::Shape>& shape) = 0;
+
+        //! Returns the number of shapes that make up this static rigid body.
+        //! @return Returns the number of shapes as a AZ::u32.
         virtual AZ::u32 GetShapeCount() { return 0; }
+
+        //! Returns a shared pointer to the requested shape index.
+        //! @param index The index of the shapes to return. Expected to be between 0 and GetShapeCount().
```

Comment:
To be accurate, the comment about index should be: "Expected to be between 0 and GetShapeCount()-1." Perhaps it'd be better to just say "Expected to be less than GetShapeCount()".